x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/exporter/http/PublishableHttpResource.java

```diff
@@ -309,6 +311,7 @@ protected void putResource(final RestClient client,
                                final Logger logger,
                                final String resourceBasePath,
                                final String resourceName,
+                               final Map<String, String> parameters,
```

Comment:
Can you change the class level `parameters` to `defaultParamaters` or change this name so this doesn't shadow the other variable.

Reply:
I changed the class level `parameters` to `defaultParameters`.